README.md

```diff
@@ -0,0 +1,156 @@
+# To Do list:Version 1.4<!---Title of Project-->
+<a name="readme-top"></a>
+
+
+## React capstone project - Metrics webapp <!---Title of Project-->
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metrics webapp<a name="about-project"></a> <!---Project title-->
+This React capstone project is about building a mobile web application to check a list of metrics that We've created making use of React and Redux. This website list the genres of popular shows and allows to see the shows descriptions.
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+ React, Redux and CSS, and linter configuration setup.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[Live Demo](moviesgenre.netlify.app)
```

Comment:
- [ ] Hey @Odongo006 🙋‍♀️ As suggested by the previous reviewers, please make sure that the live demo link is working 🛠 Right now I'm getting the following error 👇 I think the issue is with how is added in the README. Consider changing it to this 👉 `[Live Demo](https://moviesgenre.netlify.app/)`

![image](https://user-images.githubusercontent.com/106119105/233807827-453cec04-4c0c-4217-b344-04ccad4285ab.png)